apps/artemis-web/utils/power-actions.ts

```diff
@@ -0,0 +1,111 @@
+import { configure } from 'react-hotkeys';
+
+export const commands = (role) => (
+
+    role === 'admin' ? {
+        Dashboard: {
+            name: 'Dashboard',
+            shortcut: 'alt+d',
+            callback: () => (window.location.pathname = '/dashboard'),
+        },
+        BGPUpdates: {
+            name: 'BGP Updates',
+            shortcut: 'alt+b',
+            callback: () => (window.location.pathname = '/bgpupdates'),
+        },
+        Hijacks: {
+            name: 'Hijacks',
+            shortcut: 'alt+h',
+            callback: () => (window.location.pathname = '/hijacks'),
+        },
+        System: {
+            name: 'System',
+            shortcut: 'alt+s',
+            callback: () => (window.location.pathname = '/admin/system'),
+        },
+        UserManagement: {
+            name: 'User Management',
+            shortcut: 'alt+m',
+            callback: () => (window.location.pathname = '/admin/user_management'),
+        },
+        Password: {
+            name: 'Password Change',
+            shortcut: 'alt+p',
+            callback: () => (window.location.pathname = '/password_change'),
+        },
+        Config: {
+            name: 'Config Comparison',
+            shortcut: 'alt+c',
+            callback: () => (window.location.pathname = '/config_comparison'),
+        },
+    } : {
+        Dashboard: {
+            name: 'Dashboard',
+            shortcut: 'alt+d',
+            callback: () => (window.location.pathname = '/dashboard'),
+        },
+        BGPUpdates: {
+            name: 'BGP Updates',
+            shortcut: 'alt+b',
+            callback: () => (window.location.pathname = '/bgpupdates'),
+        },
+        Hijacks: {
+            name: 'Hijacks',
+            shortcut: 'alt+h',
+            callback: () => (window.location.pathname = '/hijacks'),
+        },
+        Password: {
+            name: 'Password Change',
+            shortcut: 'alt+p',
+            callback: () => (window.location.pathname = '/password_change'),
+        },
+        Config: {
+            name: 'Config Comparison',
+            shortcut: 'alt+c',
+            callback: () => (window.location.pathname = '/config_comparison'),
+        },
+    });
+
+export const keyMap = {
```

Comment:
does not seem to work on Linux.... Will check in case I am missing sth. Have you tested on gitpod or linux VM?

Reply:
I think I did on Ubuntu... But will check again now

Reply:
Hmm on Gitpod works. Did you try pressing alt + d or alt + b  to see if It changes the page?